portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoValueLocalServiceImpl.java

```diff
@@ -85,10 +83,11 @@ public ExpandoValue addValue(
 			String columnName, long classPK, boolean data)
 		throws PortalException {
 
-		ExpandoTable table = _expandoTableLocalService.getTable(
-			companyId, className, tableName);
+		ExpandoTable table = _expandoTablePersistence.findByC_C_N(
+			companyId, _classNameLocalService.getClassNameId(className),
+			tableName);
 
-		ExpandoColumn column = _expandoColumnLocalService.getColumn(
+		ExpandoColumn column = _expandoColumnPersistence.fetchByT_N(
```

Comment:
@slnn Why you do null check in some cases and don't in other cases, like this one?